packages/payload/src/index.ts

```diff
@@ -430,8 +430,7 @@ export class BasePayload {
       process.env.NODE_ENV !== 'production' &&
       process.env.PAYLOAD_DISABLE_DEPENDENCY_CHECKER !== 'true'
     ) {
-      // First load. First check if there are mismatching dependency versions of payload packages
```

Comment:
Feels like this code has gotten quite verbose, would like to see all of this logic moved into a util, so it can be called with a simple function from `src/index.ts`.

Reply:
I agree, done! Extracted it into its own file